python/lsst/ts/xml/data/sal_interfaces/MTMount/MTMount_Commands.xml

```diff
@@ -48,6 +48,109 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTMount</Subsystem>
+    <EFDB_Topic>MTMount_command_setThermal</EFDB_Topic>
+    <Description>Control one or more thermal systems.</Description>
+    <item>
+      <EFDB_Name>azimuthDrivesState</EFDB_Name>
+      <Description>Desired thermal state; a ThermalCommandState enum.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>elevationDrivesState</EFDB_Name>
+      <Description>Desired thermal state; a ThermalCommandState enum.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>cabinet0101State</EFDB_Name>
+      <Description>Desired thermal state; a ThermalCommandState enum.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>mainCabinetState</EFDB_Name>
+      <Description>Desired thermal state; a ThermalCommandState enum.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>auxiliaryCabinetsState</EFDB_Name>
+      <Description>Desired thermal state; a ThermalCommandState enum. Note that the MobBus thermal controller is always on or in fault, so mode=Off is ignored.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>oilSupplySystemCabinetState</EFDB_Name>
+      <Description>Desired thermal state; a ThermalCommandState enum. Note that the MobBus thermal controller is always on or in fault, so mode=Off is ignored.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>topEndChillerState</EFDB_Name>
+      <Description>Desired thermal state; a ThermalCommandState enum.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>azimuthDrivesSetpoint</EFDB_Name>
+      <Description>Desired temperature; ignored unless the corresponding state is On.</Description>
```

Comment:
Interesting double negative. I guess `only accepted if the corresponding state is On` or something similar would have worked as well. Like this it's fine.

Reply:
I'll change it to "only used if..."